bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/link/ItemChannelLinkRegistry.java

```diff
@@ -193,4 +218,17 @@ protected void notifyListenersAboutUpdatedElement(final ItemChannelLink oldEleme
         super.notifyListenersAboutUpdatedElement(oldElement, element);
         // it is not needed to send an event, because links can not be updated
     }
+
+    private boolean isEditable(String linkId) {
+        if (getManagedProvider().isPresent()) {
+            return getManagedProvider().get().get(linkId) != null;
+        }
+        return false;
+    }
```

Comment:
Hi @seime,
I am cleaning up some SAT warnings. This method seems unused.
Was it intended for something specific or can I safely remove it?

![grafik](https://github.com/user-attachments/assets/b9c327a8-df06-41e5-b5da-94cea11b248f)

Thanks!

Reply:
Probably a remain from merging the PR to the PR. Just remove

Reply:
ok, thanks